src/charmed_kubeflow_chisme/testing/cos_integration.py

```diff
@@ -335,3 +356,31 @@ async def assert_logging(app: Application) -> None:
         assert (
             "endpoint" in unit_data
         ), f"{APP_LOGGING} unit '{unit_name}' relation data are missing 'endpoint'"
+
+
+async def assert_grafana_dashboards(app: Application, dashboards: Set[str]) -> None:
```

Comment:
Should this have an argument for relation name that will default to APP_GRAFANA_DASHBOARD? I think it makes sense since this is configurable in theory, even if all CKF charms use the same relation name.

Reply:
Maybe it's good idea, you can still do it with changing the constant variable. e.g. the [integration tests](https://github.com/canonical/charmed-kubeflow-chisme/blob/4a1a8ab765ed0007ef7c84754c4feff551de0084/tests/integration/test_cos.py#L15-L17)
I think that it's safer to use the constant instead of argument in function, because this constant is used also during deployment and it should be same. We can open a separate issue for this, since if we go with argument, we should add it to all other function also. WDYT?

Reply:
Yes agreed on that last point. Do you think we could implement this as part of this PR (since it won't break existing integrations) instead of opening an issue? If not, I 'm OK with issue too. I jsut think it would not be a that big of a change.

Reply:
You agreed that it's safer to use constant?
I think we should open an issue and discuss it there, I can change it here, but we should also change it for other functions. e.g. `assert_logging` or `assert_metrics_endpoint`

Reply:
Opened the issue #111.